incident/urls.py

```diff
@@ -0,0 +1,56 @@
+from django.urls import path
+from . import views
+
```

Comment:
you could add **app_name = "incident"** so that you don't have to user /api/v1/incident and just also use api/v1 and include the URL patterns